tracer/test/agent/MockAgent/Options.cs

```diff
@@ -0,0 +1,45 @@
+using CommandLine;
+
+namespace MockAgent
+{
+    public class Options
+    {
```

Comment:
Is this actually used 🤔 Isn't there a more general "mock agent" project somewhere? I can see that this is useful for testing, I'm just a bit loathe to introduce _more_ projects we need to build in ci...

Reply:
I can exclude this from building in CI, but I found it to be invaluable in debugging/developing.